sphinx/cmdline.py

```diff
@@ -88,38 +88,40 @@ def get_parser():
     parser = argparse.ArgumentParser(
         usage='usage: %(prog)s [OPTIONS] SOURCEDIR OUTPUTDIR [FILENAMES...]',
         epilog='For more information, visit <http://sphinx-doc.org/>.',
+        formatter_class=argparse.RawDescriptionHelpFormatter,
         description="""
 Generate documentation from source files.
 
-sphinx-build generates documentation from the files in SOURCEDIR and places it
-in OUTPUTDIR. It looks for 'conf.py' in SOURCEDIR for the configuration
-settings.  The 'sphinx-quickstart' tool may be used to generate template files,
-including 'conf.py'
+sphinx-build generates documentation from the files in SOURCEDIR and places
+it in OUTPUTDIR. It looks for 'conf.py' in SOURCEDIR for the configuration
+settings.  The 'sphinx-quickstart' tool may be used to generate template
+files, including 'conf.py'.
 
 sphinx-build can create documentation in different formats. A format is
-selected by specifying the builder name on the command line; it defaults to
-HTML. Builders can also perform other tasks related to documentation
+selected by specifying the builder name on the command line; it defaults
+to HTML. Builders can also perform other tasks related to documentation
 processing.
 
 By default, everything that is outdated is built. Output only for selected
 files can be built by specifying individual filenames.
 """)
 
-    parser.add_argument('--version', action='version', dest='show_version',
+    parser.add_argument('--version', action='version',
                         version='%%(prog)s %s' % __display_version__)
 
     parser.add_argument('sourcedir',
                         help='path to documentation source files')
     parser.add_argument('outputdir',
                         help='path to output directory')
     parser.add_argument('filenames', nargs='*',
-                        help='a list of specific files to rebuild. Ignored '
+                        help='a list of specific files to rebuild, ignored '
                         'if -a is specified')
 
     group = parser.add_argument_group('general options')
-    group.add_argument('-b', metavar='BUILDER', dest='builder',
-                       default='html',
-                       help='builder to use (default: html)')
+    group.add_argument('-b', metavar='BUILDER', dest='builders',
+                       action='append', default='html',
```

Comment:
Could you really test this on local?
I got a following error:
```
Traceback (most recent call last):
  File ".tox/py27/bin/sphinx-build", line 11, in <module>
    sys.exit(main())
  File "/Users/tkomiya/work/sphinx/.tox/py27/lib/python2.7/site-packages/sphinx/cmd/build.py", line 38, in main
    return build_main(argv)
  File "/Users/tkomiya/work/sphinx/.tox/py27/lib/python2.7/site-packages/sphinx/cmd/build.py", line 23, in build_main
    return cmdline.main(argv)  # type: ignore
  File "/Users/tkomiya/work/sphinx/.tox/py27/lib/python2.7/site-packages/sphinx/cmdline.py", line 187, in main
    args = parser.parse_args(argv)
  File "/Users/tkomiya/.pyenv/versions/2.7.14/lib/python2.7/argparse.py", line 1701, in parse_args
    args, argv = self.parse_known_args(args, namespace)
  File "/Users/tkomiya/.pyenv/versions/2.7.14/lib/python2.7/argparse.py", line 1733, in parse_known_args
    namespace, args = self._parse_known_args(args, namespace)
  File "/Users/tkomiya/.pyenv/versions/2.7.14/lib/python2.7/argparse.py", line 1939, in _parse_known_args
    start_index = consume_optional(start_index)
  File "/Users/tkomiya/.pyenv/versions/2.7.14/lib/python2.7/argparse.py", line 1879, in consume_optional
    take_action(action, args, option_string)
  File "/Users/tkomiya/.pyenv/versions/2.7.14/lib/python2.7/argparse.py", line 1807, in take_action
    action(self, namespace, argument_values, option_string)
  File "/Users/tkomiya/.pyenv/versions/2.7.14/lib/python2.7/argparse.py", line 930, in __call__
    items.append(values)
AttributeError: 'str' object has no attribute 'append'
```

I guess this default value should be changed to array.

Reply:
Yup, that's a rebase conflict. I'll fix shortly, though I do need to fix the other issues first, heh